docs/networking/controlling-fixtures-over-a-network.md

```diff
@@ -141,31 +143,32 @@ the start of the chapter.
 4. Select the Art-Net module from the node list in the left hand side
 of the window.
 
-5. Press the small 'i' button to show the node properties. Ensure that
-the correct Network adapter is set (this may not be the Default Adapter)
+5. Press the small cog button to show the module properties. Ensure that
+the correct network adapter is set (this may not be the Default Adapter)
```

Comment:
```suggestion
5. Press the small cog button to show the module properties. Ensure that
the correct network adapter is set (this may not be the default adapter).
```